src/buildHTML.js

```diff
@@ -1427,31 +1434,52 @@ groupTypes.accent = function(group, options) {
         style.metrics.xHeight);
 
     // Build the accent
-    const accent = buildCommon.makeSymbol(
-        group.value.accent, "Main-Regular", "math", options);
-    // Remove the italic correction of the accent, because it only serves to
-    // shift the accent over to a place we don't want.
-    accent.italic = 0;
-
-    // The \vec character that the fonts use is a combining character, and
-    // thus shows up much too far to the left. To account for this, we add a
-    // specific class which shifts the accent over to where we want it.
-    // TODO(emily): Fix this in a better way, like by changing the font
-    const vecClass = group.value.accent === "\\vec" ? "accent-vec" : null;
-
-    let accentBody = makeSpan(["accent-body", vecClass], [
-        makeSpan([], [accent])]);
-
-    accentBody = buildCommon.makeVList([
-        {type: "elem", elem: body},
-        {type: "kern", size: -clearance},
-        {type: "elem", elem: accentBody},
-    ], "firstBaseline", null, options);
+    let accentBody;
+    if (!group.value.isStretchy) {
+        const accent = buildCommon.makeSymbol(
+            group.value.label, "Main-Regular", "math", options);
+        // Remove the italic correction of the accent, because it only serves to
+        // shift the accent over to a place we don't want.
+        accent.italic = 0;
+
+        // The \vec character that the fonts use is a combining character, and
+        // thus shows up much too far to the left. To account for this, we add a
+        // specific class which shifts the accent over to where we want it.
+        // TODO(emily): Fix this in a better way, like by changing the font
+        const vecClass = group.value.label === "\\vec" ? "accent-vec" : null;
+
+        accentBody = makeSpan(["accent-body", vecClass], [
+            makeSpan([], [accent])]);
+
+        // Shift the accent over by the skew. Note we shift by twice the skew
+        // because we are centering the accent, so by adding 2*skew to the left,
+        // we shift it to the right by 1*skew.
+        accentBody.style.marginLeft = 2 * skew + "em";
+
+        accentBody = buildCommon.makeVList([
+            {type: "elem", elem: body},
+            {type: "kern", size: -clearance},
+            {type: "elem", elem: accentBody},
+        ], "firstBaseline", null, options);
+
+    } else {
+        const ruleWidth = fontMetrics.metrics.defaultRuleThickness /
+            style.sizeMultiplier;
 
-    // Shift the accent over by the skew. Note we shift by twice the skew
-    // because we are centering the accent, so by adding 2*skew to the left,
-    // we shift it to the right by 1*skew.
-    accentBody.children[1].style.marginLeft = 2 * skew + "em";
+        accentBody = stretchy.svgSpan(group, options);
+
+        if (skew > 0) {
+            // Nudge the accent to the right.
+            const adjSize = "calc(100% - " + 2 * skew + "em) 100%";
+            accentBody.style.backgroundSize = adjSize;
```

Comment:
That line of code did not shorten the image. So I changed it. The current line includes an extra pair of parens to emphasize the correct  operator precedence: `"calc(100% - " + (2 * skew) + "em) 100%"`